paasta_tools/metrics/metastatus_lib.py

```diff
@@ -78,8 +79,20 @@ def get_mesos_disk_status(metrics):
     return total, used, available
 
 
+def get_mesos_gpu_status(metrics):
+    """Takes in the mesos metrics and analyzes them, returning gpus status.
+
+    :param metrics: mesos metrics dictionary
+    :returns: Tuple of the output array and is_ok bool
+    """
+    total = metrics['master/gpus_total']
```

Comment:
Are these available in master right now? I think mesos only adds it if we have slaves with gpus in the cluster. It won't have it otherwise.  Let's check if this keys is available or not.